tests/Functional/Fixtures/Controller/TagController.php

```diff
@@ -69,6 +69,6 @@ public function subrequestAction()
 
     public function twigAction()
     {
-        $this->render('::tag.html.twig');
+        return $this->render('::tag.html.twig');
```

Comment:
it probably means the test does not check the returned response as that will be a 500 status...